integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -26,6 +26,10 @@
 {{- if $isSingleMaster }}
   {{- $alluxioJavaOpts = printf "-Dalluxio.master.hostname=%v-%v" $fullName $defaultMasterName | append $alluxioJavaOpts }}
 {{- end }}
+{{ if .Values.fuse.enabled -}}
+{{- $alluxioJavaOpts = print "-Dalluxio.user.hostname=${ALLUXIO_CLIENT_HOSTNAME}" | append $alluxioJavaOpts }}
+{{- $alluxioJavaOpts = printf "-Dalluxio.worker.hostname=${ALLUXIO_CLIENT_HOSTNAME}" | append $alluxioJavaOpts }}
```

Comment:
@madanadit I'm not too sure about what java opts Fuse wants. But it can be related to this `ALLUXIO_CLIENT_JAVA_OPTS` not being used https://github.com/Alluxio/alluxio/blob/db1418edcdefc0723f436ec1353843377ea3a694/integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml#L96

Looking at https://github.com/Alluxio/alluxio/blob/db1418edcdefc0723f436ec1353843377ea3a694/integration/docker/entrypoint.sh#L30, I think it should be `ALLUXIO_USER_JAVA_OPTS` instead of `ALLUXIO_CLIENT_JAVA_OPTS`? So by changing https://github.com/Alluxio/alluxio/blob/db1418edcdefc0723f436ec1353843377ea3a694/integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml#L96 from `ALLUXIO_CLIENT_JAVA_OPTS` to `ALLUXIO_USER_JAVA_OPTS`, we can fix this issue without adding "-Dalluxio.worker.hostname=${ALLUXIO_CLIENT_HOSTNAME}"

Reply:
I've tried, no help.

Reply:
We put it in alluxioJavaOpts for a long time, and it was working fine. I suggest you should test it when making code reconstruction.